lib/mcb/commands/users/list.rb

```diff
@@ -0,0 +1,16 @@
+summary 'List users in the tb'
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+
+  users = if args.any?
+            User.where(id: args.to_a)
+          else
+            User.all
```

Comment:
Whats the use case for getting all users listed?

Reply:
I imagine it would be useful for searching for a user if you weren't sure of their userid, for example, just pass the output into less and you can search it easily, etc. Or even just to test that things are working ...

Reply:
Just a thought, you're not mixing up the `list` and `show` commands are you?

Reply:
👍